github/Artifact.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2020 Yannick Jadoul <yannick.jadoul@belgacom.net>                  #
+# Copyright 2020 Jacek Pliszka <JacekPliszka@github>                           #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+import github.GithubObject
+
+
+class Artifact(github.GithubObject.CompletableGithubObject):
```

Comment:
Hi, before seeing your PR, I also did something similar and [created a PR](https://github.com/PyGithub/PyGithub/pull/1985). Please take a look and see if there are any useful parts that you could incorporate like some of the tests...